app/src/main/java/com/infomaniak/mail/utils/Extensions.kt

```diff
@@ -620,3 +622,7 @@ private fun Spannable.setClickableSpan(startIndex: Int, endIndex: Int, onClick:
         Spannable.SPAN_EXCLUSIVE_EXCLUSIVE,
     )
 }
+
+fun MaterialButton.showProgressAfterTimer() {
+    RUtils.createRefreshTimer(onTimerFinish = ::showProgress).start()
```

Comment:
Shouldn't this be "CoreUtils" rather than "RUtils"? Because it has nothing to do with R